src/app/+run-tale/run-tale/modals/register-data-dialog/register-data-dialog.component.html

```diff
@@ -25,9 +25,9 @@ <h1 mat-dialog-title>Select Data</h1>
           <div class="info-block">
             <p>
               The URL or DOI of the data object. Data packages can be imported into WholeTale from 
-              <a href="https://www.dataone.org">DataONE</a> and select <a href="https://www.globus.org">Globus</a> 
+	      <a href="https://www.dataone.org">DataONE</a>, <a href="https://dataverse.org/">Dataverse</a>, <a href="https://zenodo.org/">Zenodo</a> and select <a href="https://www.globus.org">Globus</a> 
```

Comment:
We probably want these to open in a new window too
```suggestion
	      <a href="https://www.dataone.org" target="_blank">DataONE</a>, <a href="https://dataverse.org/" target="_blank">Dataverse</a>, <a href="https://zenodo.org/" target="_blank">Zenodo</a> and select <a href="https://www.globus.org" target="_blank">Globus</a> 
```